cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -3160,19 +3183,15 @@ pDRepHashSource =
 
 pDRepScriptHash :: Parser ScriptHash
 pDRepScriptHash =
-  Opt.option scriptHashReader $ mconcat
-    [ Opt.long "drep-script-hash"
-    , Opt.metavar "HASH"
-    , Opt.help "DRep script hash (hex-encoded). Obtain it with \"cardano-cli conway governance hash script ...\"."
-    ]
+  pScriptHash
+    "drep-script-hash"
+    "DRep script hash (hex-encoded). Obtain it with \"cardano-cli conway governance hash script ...\"."
 
 pConstitutionScriptHash :: Parser ScriptHash
 pConstitutionScriptHash =
-  Opt.option scriptHashReader $ mconcat
-    [ Opt.long "constitution-script-hash"
-    , Opt.metavar "HASH"
-    , Opt.help "Constitution script hash (hex-encoded). Obtain it with \"cardano-cli conway governance hash script ...\"."
-    ]
+  pScriptHash
+    "constitution-script-hash"
+    "Constitution script hash (hex-encoded). Obtain it with \"cardano-cli conway governance hash script ...\"."
```

Comment:
conway governance hash anchor-data...

Reply:
@CarlosLopezDeLara> I'm not sure what you mean. This help text seems correct (it is about the hash of a script).